pkg/controller/plan/kubevirt.go

```diff
@@ -1304,17 +1304,12 @@ func (r *KubeVirt) getOsMapConfig(providerType v1beta1.ProviderType) (configMap
 
 func (r *KubeVirt) getPreference(vm *plan.VMStatus, preferenceName string) (name, kind string, err error) {
 	name, kind, err = r.getVirtualMachinePreference(preferenceName)
-	if err != nil {
-		if k8serr.IsNotFound(err) {
-			r.Log.Info("could not find a local instance type preference for destination VM. trying cluster wide",
-				"vm",
-				vm.String())
-		} else {
-			return
-		}
+	if err != nil && k8serr.IsNotFound(err) {
```

Comment:
wouldn't it make sense to always fall back to the cluster's preference in case of an error? (sure, when there's no error we shouldn't proceed to getting the cluster's preference)

Reply:
i think that once we see an error we should return, clearly something bad happened.
we can also say: preferences aren't so important to our point of view, then try our best and fallback.

i prefer the first option

Reply:
> ..., clearly something bad happened

this "bad" thing might be lack of permissions for getting the preference in the target namespace..
if we say that we prioritized namespaced preferences first, then cluster preferences and then templates - the question is why to fall back from namespaced preferences to templates when we face an issue in querying the namespaced preference